src/Components/calculator.js

```diff
@@ -1,69 +1,59 @@
 import './calculator.css';
-import React from 'react';
+import React, { useState } from 'react';
 import Button from './Button';
 import calculate from './logic/calculate';
-// import operate from './logic/operate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      operation: null,
-      next: null,
-    };
-    this.updateCalc = this.updateCalc.bind(this);
-  }
+function Calculator() {
```

Comment:
- [ ] [OPTIONAL] we can make this component function into an ES6 by using an arrow function.
```
const Calculator = () => {
# code here
};
```

Reply:
thank you @elisha2kyakpo1 , I am working on it now.